api_tests/nodes/serializers/test_serializers.py

```diff
@@ -1,64 +1,62 @@
+class TestNodeSerializer:
 
-    def setUp(self):
-        super(TestNodeSerializer, self).setUp()
-        self.user = UserFactory()
+    def test_node_serializer(self, user):
 
-    def test_node_serialization(self):
-        parent = ProjectFactory(creator=self.user)
-        node = NodeFactory(creator=self.user, parent=parent)
+    #   test_node_serialization
+        parent = ProjectFactory(creator=user)
+        node = NodeFactory(creator=user, parent=parent)
         req = make_drf_request_with_version(version='2.0')
         result = NodeSerializer(node, context={'request': req}).data
         data = result['data']
-        assert_equal(data['id'], node._id)
-        assert_equal(data['type'], 'nodes')
+        assert data['id'] == node._id
+        assert data['type'] == 'nodes'
 
         # Attributes
         attributes = data['attributes']
-        assert_equal(attributes['title'], node.title)
-        assert_equal(attributes['description'], node.description)
-        assert_equal(attributes['public'], node.is_public)
-        assert_equal(attributes['tags'], [str(each.name) for each in node.tags.all()])
-        assert_equal(attributes['current_user_can_comment'], False)
-        assert_equal(attributes['category'], node.category)
-        assert_equal(attributes['registration'], node.is_registration)
-        assert_equal(attributes['fork'], node.is_fork)
-        assert_equal(attributes['collection'], node.is_collection)
+        assert attributes['title'] == node.title
+        assert attributes['description'] == node.description
+        assert attributes['public'] == node.is_public
+        assert attributes['tags'] == [str(each.name) for each in node.tags.all()]
+        assert attributes['current_user_can_comment'] == False
+        assert attributes['category'] == node.category
+        assert attributes['registration'] == node.is_registration
+        assert attributes['fork'] == node.is_fork
+        assert attributes['collection'] == node.is_collection
 
         # Relationships
         relationships = data['relationships']
-        assert_in('children', relationships)
-        assert_in('contributors', relationships)
-        assert_in('files', relationships)
-        assert_in('parent', relationships)
-        assert_in('affiliated_institutions', relationships)
+        assert 'children' in relationships
+        assert 'contributors' in relationships
+        assert 'files' in relationships
+        assert 'parent' in relationships
+        assert 'affiliated_institutions' in relationships
         parent_link = relationships['parent']['links']['related']['href']
-        assert_equal(
-            urlparse(parent_link).path,
-            '/{}nodes/{}/'.format(API_BASE, parent._id)
-        )
-        assert_in('registrations', relationships)
+        assert (
+            urlparse(parent_link).path ==
+            '/{}nodes/{}/'.format(API_BASE, parent._id))
+        assert 'registrations' in relationships
```

Comment:
You're saying to change the assert 'registrations' in relationships to assert * in relationships?